plugins/advisors/osv/src/main/kotlin/Osv.kt

```diff
@@ -182,43 +182,48 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest? {
 }
 
 private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.vulnerabilities.Vulnerability {
-    // OSV uses a list in order to support multiple representations of the severity using different scoring systems.
```

Comment:
commit message: typo in "unlcear"

Reply:
Fix, plus a few more typos I've found when proof-reading again.